README.md

```diff
@@ -1,26 +1,104 @@
-# ember-table
+# Ember Table
 
-This README outlines the details of collaborating on this Ember addon.
+An addon to support large data set and a number of features around table. `Ember Table` can
+handle over 100,000 rows without and rendering or performance issue.
 
-## Installation
+## Install
 
-* `git clone <repository-url>` this repository
-* `cd ember-table`
-* `npm install`
+```bash
+ember install ember-table
+```
 
-## Running
+## Features
+- Column resizing, column reordering.
+- Table resizing.
+- Fixed first column.
+- Custom row and custom header.
+- Handles transient state at cell level.
+- Single, multiple row selection.
+- Table grouping.
 
-* `ember serve`
-* Visit your app at [http://localhost:4200](http://localhost:4200).
+## Usage.
 
-## Running Tests
+To use `Ember Table`, you need to create `columns` and `rows` dataset.
 
-* `npm test` (Runs `ember try:each` to test your addon against multiple Ember versions)
-* `ember test`
-* `ember test --server`
+`columns` is an array of `ColumnDefinition` which has multiple fields to define behavior of column.
+Two required field for `ColumnDefinition` is `columnName` and `valuePath`.
 
-## Building
+```javascript
+  columns: [
+    ColumnDefinition.create({
```

Comment:
Just POJOs, we shouldn't enforce the ColumnDefinition class. There's no need for it.

Reply:
Right now, it's based on `ColumnDefinition`. We have not found a good solution with using POJO because we want to listen to column width. That's why I keep it for now. We can change it to POJO if needed in the beta version

Reply:
You can listen to property changes on POJOs, as long as you use `get/set` which is standard in Ember. The only thing you can't listen to directly in 1.11 are changes to array length, which can be solved by re-setting the array after changes.